software/deluge.yml

```
@@ -0,0 +1,11 @@
+name: Deluge
+website_url: https://deluge-torrent.org/
+description: Deluge is a lightweight, Free Software, cross-platform BitTorrent client.
```

```suggestion
description: Lightweight, cross-platform BitTorrent client.
```

Minor fix to be more consistent with other entries.